cmd/pbm/main.go

```diff
@@ -33,6 +33,7 @@ var (
 	configShowKey       = configCmd.Arg("key", "Show the value of a specified key").String()
 
 	backupCmd      = pbmCmd.Command("backup", "Make backup")
+    bcpAdminTag    = backupCmd.Flag("tagname", "Set Tag for preferred Secondary <name=value>").StringMap()
```

Comment:
```suggestion
    bcpTags    = backupCmd.Flag("tag", "Tags for preferred secondary nodes <name=value>").StringMap()
```

Reply:
Ok for suggest, but it broke this code, I will implement your solution

`storeString, err := backup(pbmClient, bcpName, *bcpCompression, *bcpAdminTag)`